docker/Dockerfile.cpu

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11
+
+RUN apt-get update && apt-get install -y \
+	ffmpeg
+
+WORKDIR /app
+
+ADD requirements.txt .
+RUN pip install -r requirements.txt
+
+ADD inswapper_128.onnx .
+ADD run.py .
+ADD roop roop
+
+ENV HOME=/cache
+
+RUN mkdir /cache
+
+ENTRYPOINT ["python", "/app/run.py"]
```

Comment:
ENTRYPOINT ["python", "run.py"

Reply:
First you say `WORKDIR /app` is not needed, then you say `ENTRYPOINT ["python", "run.py"]`. How should Python find `run.py` without the WORKDIR or without the absolute path.
As far as I remember, Python says "explicit is better than implicit".